cmd/root.go

```diff
@@ -73,7 +74,16 @@ func initConfig() {
 
 	viper.AutomaticEnv()
 
-	if err := viper.ReadInConfig(); err == nil {
+	err := viper.ReadInConfig()
+
+	var notFound *viper.ConfigFileNotFoundError
```

Comment:
Not an easy one for me to find at first but this will actually cause a panic. I haven't used the `errors.As(...)` much but I tested it out locally and got this:
```
panic: errors: target must be a non-nil pointer

goroutine 1 [running]:
errors.As({0x14b5740, 0xc0001c86e0}, {0x140ce20, 0x0})
        /usr/local/go/src/errors/wrap.go:84 +0x425
github.com/spf13/cobra-cli/cmd.initConfig()
        /Users/jschnake/go/src/github.com/spf13/cobra-cli/cmd/root.go:81 +0x18e
```

The issue is that the error you create needs to be non-nil. You created a pointer with a type but not a value. You apparently need to create an actual instance of that error type
```
	notFound:= &viper.ConfigFileNotFoundError{}
```

Reply:
Updated : )